scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -1318,6 +1331,11 @@ def gpio_hogs(self) -> List[ControllerAndData]:
 
         return res
 
+    @property
+    def has_child_binding(self) -> bool:
+        "True if the node can have child nodes, False otherwise"
```

Comment:
I'm not sure that this description is accurate, a node can have child nodes even if child-binding is not declared in binding schema

Reply:
Thanks for your review. :-) You are of course right. Fixed.